eth/downloader/downloader.go

```diff
@@ -18,61 +18,64 @@
 package downloader
 
 import (
-	"bytes"
 	"errors"
 	"math"
 	"math/big"
-	"math/rand"
 	"sync"
 	"sync/atomic"
 	"time"
 
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/core"
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/event"
 	"github.com/ethereum/go-ethereum/logger"
 	"github.com/ethereum/go-ethereum/logger/glog"
-	"gopkg.in/fatih/set.v0"
 )
 
 const (
-	eth60 = 60 // Constant to check for old protocol support
-	eth61 = 61 // Constant to check for new protocol support
+	eth61 = 61 // Constant to check for old protocol support
```

Comment:
(2) Ah, I see. The versions are 60/61/62/63/64 decimal. I think that post release we can clean up the POC series from the wiki page but will get the original authors' consent too :) Thanks for pointing it out :)